docs/experience/memory-usage.md

```diff
@@ -0,0 +1,39 @@
+title: Monitoring Browser Memory Usage
+description: Gathering information about web page memory usage
+
+Browsers manage the memory used by web pages automatically without user interaction. Sematext Experience allows monitoring the memory usage for your web pages to identify issues, regressions, and give you an estimate of the memory needed by your applications. 
+
+<img
+  class="content-modal-image"
+  alt="Memory Usage Overview"
+  title="Memory Usage Overview"
+  src="../../images/experience/memory/overview.png"
+/>
+
+## Use Cases
+There are many use cases where having the view of the browser memory usage can be crucial to identify and fix problems related as soon as possible. Those include:
```

Comment:
There are many use cases where having the view of the browser memory usage can be crucial to identify and fix problems related as soon as possible.
=>
There are many use cases where having insight into the browser's memory usage can be crucial to identify and fix problems as soon as possible.

Reply:
Adjusted.